server/src/sync.ts

```diff
@@ -38,7 +38,16 @@ export async function initSync(id: string, syncOptions: SyncOptions) {
       continue;
 
     for (const phoneNumber of googleContact.numbers) {
-      const whatsappContactId = whatsappContacts.get(phoneNumber);
+      let whatsappContactId = whatsappContacts.get(phoneNumber);
+
+      // Fix for Brazilian numbers with extra '9'
+      if (!whatsappContactId && phoneNumber.slice(0, 2) === '55') {
```

Comment:
Make sure to run Prettier on the code (since I'm mostly the only dev it's not automated yet)

Reply:
OK, I'll run Prettier and try to make the code better, trying to avoid the double !whatsappContactId check and your other comments.